docker/docker-compose.yml

```diff
@@ -157,3 +185,22 @@ services:
         hard: -1
     ports:
       - '9200'
+    labels:
+      # Enable Traefik in development mode
+      - 'traefik.enable=true'
+      # Traefik routing for elasticsearch at http://elasticsearch.localhost
+      - 'traefik.http.routers.elasticsearch.rule=Host(`elasticsearch.localhost`)'
+      # Specify the redis port
+      - 'traefik.http.services.elasticsearch.loadbalancer.server.port=9200'
+
+  apm:
```

Comment:
This belongs in production.yml

Reply:
According to [This ToDo](https://github.com/Seneca-CDOT/telescope/blob/32581ed2548e392e2ec98d560459fe73b8c5473e/docker/production.yml#L263), I will move the APM part to production.yml.

Reply:
We haven't done the Elastic/ELK stuff yet for logging, etc.  We started to, but will follow-up with it later.